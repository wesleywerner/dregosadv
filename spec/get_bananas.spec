superbrief
gonear garden portal
examine trees
examine bananas
take banana
whip tree
examine banana
whip banana
eat banana
whip banana
take banana
eat it
