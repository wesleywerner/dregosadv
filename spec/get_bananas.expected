  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 200410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
  You are in a canoe on the Maro river. Your guide is with you, rowing the
  craft.
. 
> >  Dr Ego and the Egg of Man-Toomba is now in its "superbrief" mode, which
  always gives short descriptions of locations (even if you haven't been
  there before).
  
> >  Garden
  
  A small hut is to the northeast.
  
  On the fruit trees is a banana.
  
> >  You see a cherry tree but it is not bearing any fruit, the banana tree next
  to it however has a bunch of fruit.
  
> >  A bunch of wild bananas hang from the top of the tree.
  
> >  The bananas are way up in the tree, out of reach.
  
> >  You take careful aim at the bunch of bananas up in the tree, swing your arm
  in a smooth arch and *SNAP* -- a single banana drops to the ground.
  
> >  It is a wild banana with a bright green and yellow skin.
  
> >  You don't need another banana right now.
  
> >  (first taking the banana)
  You peel the banana and take a bite, the fruit is firm and sweet with a
  subtle flavor. You finish eating the rest of the delicious fruit - You feel
  revitalised.
  
> >  *SNAP* -- Another banana drops to the ground. Years of using your whip is
  finally paying off.
  
> >  Taken.
  
> >  You have had your fill of bananas.
  
> >